src/yang/ietf-bgp.yang

```diff
@@ -281,11 +278,16 @@ module ietf-bgp {
 
         list neighbor {
           key "remote-address";
-
           description
             "List of BGP neighbors configured on the local system,
              uniquely identified by remote IPv[46] address.";
 
+          leaf remote-address {
```

Comment:
What's the motivation to move this up?  I realize that it's the key.  The motivation for the current ordering is tcp sessions are typically pictured in the sense of local:remote.

Reply:
The motivation is because of the fact that just as 'name' is the key for neighbor and appears first, 'remote-address' is key for peer-group and appears first. Also note that peer-group has plenty of other parameters that are not present in neighbor, so the order is not going to be perfect.

Reply:
This comment says other way around. Neighbor is indexed by IP address and peer-group is indexed by a name.

Reply:
Updated the comment.